### PR TITLE
Prep for B01Y23

### PR DESCRIPTION
**Pull request by harigopal**

### CODE REVIEW

File: introduction-to-nodejs/github-how-to/README.md [hunk 1]
```diff
@@ -1,55 +1,63 @@
...
+If I run `git log`, to see a log of all the commits in the repo, I can see that the commit is there, along with its message.
 
-Treat the token that is created like a password. This will be shown only once, so save it in a secure location - if you use a password manager, this string should go in there. I'll copy this string by hitting the copy button next to it.
+Now that we have a git repo with with a commit, we can push this repository to GitHub so that others can see our work. We can do this using the `git push` command. Before we can do that, we'll need to a new _empty_ repository in GitHub with the same name.
```
Comment (reenas) [inline introduction-to-nodejs/github-how-to/README.md:37]: _with_ is repeated twice in the first sentence.

File: introduction-to-nodejs/github-how-to/README.md [hunk 2]
```diff
@@ -1,55 +1,63 @@
...
+If we try `git status` again, it tells us that there is a change that is ready to be committed, and it lists the two files in the folder.
 
-Now that we have a git repo with with a commit, we can push this repository to Github so that others can see our work. You should create a new repository in Github with the same name.
+Before we make our first commit, I want to do one more thing. In the `wd201` directory, I'm going to create a file called `.gitignore`, and in this new file I'm going to write one line for now - `node_modules`. In the next couple of lessons, we're going to be writing names of certain packages we want to download in `package.json`. When NPM - the Node Package Manager - downloads these libraries, it keeps them inside a `node_modules` folder. We don't want git to track these third-party libraries. We only want git to track the history of files that _we_ create. Hence, we're telling git to ignore these `node_modules` folders.
 
-I'll create a repo in my account with the name `wd-hello-node`.
+Now that we have these three files staged for a commit, we can make our first commit.
```
Comment (reenas) [inline introduction-to-nodejs/github-how-to/README.md:27]: Phrase _staged for a commit_ is introduced here. Do we want to explain staged in a sentence?

File: introduction-to-nodejs/github-how-to/README.md (hunk 1 above)
Comment (reenas) [inline introduction-to-nodejs/github-how-to/README.md:37]: "...we'll need to **create** a new _empty_ repository..."

File: introduction-to-nodejs/github-how-to/README.md [hunk 3]
```diff
@@ -1,55 +1,63 @@
...
+If I run `git log`, to see a log of all the commits in the repo, I can see that the commit is there, along with its message.
 
-Treat the token that is created like a password. This will be shown only once, so save it in a secure location - if you use a password manager, this string should go in there. I'll copy this string by hitting the copy button next to it.
+Now that we have a git repo with with a commit, we can push this repository to GitHub so that others can see our work. We can do this using the `git push` command. Before we can do that, we'll need to a new _empty_ repository in GitHub with the same name.
 
-Now I can switch back to the terminal, and type in my username. When it asks for the password, I'll simply paste the copied value. Note that it won't show up on-screen - it's a common practice for password inputs to not show anything on-screen so as to avoid anyone looking over your shoulder from seeing your passwords. So just paste in the token, and press _Enter_.
+So, I'll go ahead create a repo in my account with the same name as the directory we created - `wd201`.
```
Comment (reenas) [inline introduction-to-nodejs/github-how-to/README.md:39]: Note that the name of the repo **should be** the same as your local directory.

File: introduction-to-nodejs/github-how-to/README.md [hunk 4]
```diff
@@ -1,55 +1,63 @@
...
 
-Indeed, if I check the repository on Github, the update has showed up there.
+Now I can switch back to the terminal, and type in my username. When it asks for the password, I'll simply paste the copied value. Note that it won't show up on-screen - it's a common practice for password inputs to not show anything on-screen so as to avoid anyone looking over your shoulder from seeing your passwords. So just paste in the token, and press _Enter_.
 
-That was a quick demo of how to push code to Github. In this course, you're going to find many tasks that ask you submit your code as Github URLs. When that happens you'll want to push your code to a Github repo, and then share the URL to that repository or a folder within the repo, with us as a part of your submission.
+Now if we go to the repository on GitHub, and reload the page, we can see that the `hello-world` folder containing `package.json` and `index.js`, and the `.gitignore` file in the root of the course repo are all now on Github. Now that we've connected these two repositories - the one on your system, and the one here in GitHub, it's much easier to push changes to the repository.
```
Comment (reenas) [inline introduction-to-nodejs/github-how-to/README.md:53]: "...`.gitignore` file in the root of the course repo are all now on Github." - change Github to GitHub.

File: introduction-to-nodejs/github-how-to/README.md [hunk 5]
```diff
@@ -1,55 +1,63 @@
...
-In our projects, we will have some folders or files that we don't want to be tracked or versioned. We can do this by creating a `.gitignore` file and add the path to folder or file that we don't need to be tracked.
+For example, if I make a small change to the `index.js` file. I'll change the console log message to `Hello GitHub!`. Now, if I run `git status`, git has detected the change. So I can `git add` the `index.js` file again, and commit the change: `git commit -m "Console log message changed"`. Now, all I need to run is `git push`, and that'll push the change to GitHub.
 
-For example, `node_modules` folder is usually excluded from git repository. To do that we will have the following content in `.gitignore` file.
+Indeed, if I check the repository on GitHub, the update has showed up there.
 
-```
-node_modules
-```
+That was a quick demo of how to create a git repo, and how push code to GitHub. In this course, you're going to find many tasks that ask you submit your code as GitHub URLs. For the most part, you're going to be using this same repository to keep _all_ of the code you write as a part of this course.
```
Comment (reenas) [inline introduction-to-nodejs/github-how-to/README.md:61]: Include few sentences on why they should use only command line to work between local directory and GitHub repo. They should not use and edit repo from browser directly, with a reason why?

File: introduction-to-nodejs/github-how-to/README.md (hunk 3 above)
Comment (harigopal) [reply introduction-to-nodejs/github-how-to/README.md:39]: It doesn't, actually... 😅 I'll explain this a tiny bit more in the script.

**Pull request merged**